federation/.env.dist.template

```diff
@@ -0,0 +1,17 @@
+LOG_LEVEL=$LOG_LEVEL
+PORT=$PORT
+GRAPHIQL=$GRAPHIQL
+
+# Database
+DB_PORT=$DB_PORT
+DB_DATABASE=$DB_DATABASE
+
+COMMUNITY_NAME=$COMMUNITY_NAME
+COMMUNITY_URL=$COMMUNITY_URL
+COMMUNITY_DESCRIPTION=$COMMUNITY_DESCRIPTION
+
+# Federation
+FEDERATION_API=$FEDERATION_API
+FEDERATION_PORT=$FEDERATION_PORT
```

Comment:
remove, its transmitted via commandline in your setup script

Reply:
please remember our last meeting last tuesday, were moritz and you will require NO commandline arguments and the api-version and port have to be defined in the configuration files.
This is the result of your requirements, so i will not flipflop the code week by week.

Reply:
Those values are written via commandline - else your nginx/pm2 script does not work. I agree with this solution, so please remove these misleading config values - they are overwritten by your `export ...` statement in the `start.sh` script.

If I am not understanding something please tell.

Reply:
i understand a 'commandline parameter' that you set the parameter and value on the commandline like you do it with the command:
./start.sh "branch"

the mentioned properties are never set per commandline, they are set inside the start.sh script by evaluating the property FEDERATION_COMMUNITY_APIS.
If you want to start a federation-module without using the start.sh-script locally, you have to define these two properties in your .env-file
the src/config/index.ts expect these properties, so thatÄs the reason why they are defined in the template-file too.

Reply:
Read below, that is not whats happening here. The values in this files are ignored but set via environment variables.
You can simply try by removing those two lines - your program will still work as before.